lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2709,17 +2717,30 @@ static SILValue createHostReceive(SILBuilder &B, SILLocation loc,
   // %4 = apply %2<Float>(..., %1, %3)
   auto tensorId = createIntValue(idNumber, B, loc);
 
+  LLVM_DEBUG(llvm::dbgs() << "Creating host receive with valueTy: ");
+  LLVM_DEBUG(valueTy.print(llvm::dbgs()));
```

Comment:
Both `SILType` and `Type` have `operator<<`.

Reply:
Done.